microsoft-edge/extensions-chromium/publish/api/using-addons-publish-api.md

```diff
@@ -0,0 +1,220 @@
+---
+description: REST endpoints to automate publishing updates to add-ons that are submitted to the Microsoft Edge Add-ons website.
+title: Using the Microsoft Edge Add-ons Publish API
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 08/10/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, publish api, add-ons publish api
+---
+# Using the Microsoft Edge Add-ons Publish API
+
+<!-- find and resolve: tbd, html comments, https://addons.edge.microsoft.com/api -->
+
+This article, along with the [Add-ons Publish API Reference][AddonsPublishAPIRef], provides an overview of the proposed Microsoft Edge Add-ons Publish API.  We look forward to your suggestions and feedback on the proposed API contracts.  Please submit your feedback at (_link TBD_).
+
+The Microsoft Edge Add-ons Publish API provides a set of REST endpoints for programmatically publishing updates to add-ons submitted to the Microsoft Edge Add-ons website.  You can use these REST endpoints to automate the process of uploading and publishing add-ons into the Microsoft Edge Add-ons website.
+
+The tentative date for the availability of the APIs is December 2021.
+
+
+<!-- ====================================================================== -->
```

Comment:
These are quick placeholder definitions. Please check these definitions.

Reply:
This is ok.